Formula/ccfits

```diff
@@ -0,0 +1,23 @@
+class Ccfits < Formula
+  desc "c++ fits api"
```

Comment:
`object oriented interface to the cfitsio library` would be a better description

Reply:
modified